src/routes/amt/index.ts

```diff
@@ -25,6 +26,7 @@ amtRouter.get('/log/event/:guid', eventLog)
 amtRouter.get('/generalSettings/:guid', generalSettings)
 amtRouter.get('/hardwareInfo/:guid', hardwareInfo)
 amtRouter.post('/power/action/:guid', powerActionValidator(), powerAction)
+amtRouter.post('/power/action/extended/:guid', powerActionExtendedValidator(), powerActionExtended)
```

Comment:
what do you think of `/power/boot/:guid` ? or `/power/bootoptions/:guid`? @matt-primrose @graikhel-intel

Reply:
I think /power/bootoptions/:guid makes more sense